app/Models/Event.php

```diff
@@ -528,8 +532,10 @@ private static function beatmapsetParams($beatmapset)
     private static function beatmapParams($beatmap, $ruleset)
     {
         $url = e(route('beatmaps.show', ['beatmap' => $beatmap, 'ruleset' => $ruleset], false));
```

Comment:
probably doesn't really matter much but this shouldn't be escaped for the `_clean` variant either...?